test/e2e/catalog_e2e_test.go

```diff
@@ -912,7 +912,7 @@ var _ = Describe("Catalog", func() {
 		require.NoError(GinkgoT(), err, "error awaiting registry pod")
 
 		subChecker := func(sub *v1alpha1.Subscription) bool {
-			return sub.Status.InstalledCSV == "etcdoperator.v0.9.2-clusterwide"
+			return sub.Status.State == v1alpha1.SubscriptionStateAtLatest
```

Comment:
I think this still needs to check installed CSV. AtLatest just means that we're at the latest version we know about, it doesn't verify that OLM has seen the newest version in the catalog

Reply:
we do the check further down at the end of the test still right?

Reply:
https://github.com/operator-framework/operator-lifecycle-manager/pull/1444/files#diff-c8a24dc59e4efc3ed8ec9b48db6c5559R933

Reply:
we could change that to "nginx.v2.0.0" though if i think that's what you're suggesting

Reply:
Yes, but there's no polling on that check. That means this could flake and fail (because OLM hasn't seen the updated catalog yet, but the subscription is at latest, and we check the csv version and fail because it's old)

Reply:
ok gotcha, will update 👍